### DISCUSSION TITLE
How to update homebrew source to the latest version?

### DISCUSSION THREAD

**Discussion by Shawshank01** [c0re100/qBittorrent-Enhanced-Edition#393] [Q&A] [answered]
I find the tracker bug was fixed in the latest version in GitHub but not in Homebrew, and I waited nearly two weeks, it's still shows [4.4.3.11].

Is there any chance I can update to the latest version using Homebrew, or I have to wait to official update? Thanks.

**Comment by c0re100** [comment, marked as answer]
You need to wait someone submit 4.4.3.12 PR to Homebrew :')

**Reply by p-linnane** [reply]
Hello! v4.4.3.12 does not appear to have a macOS .dmg file in the release. This is why it has not been updated in homebrew-cask yet. Once the .dmg is released I can submit the PR to get it updated for @Shawshank01.

**Reply by c0re100** [reply]
> Hello! v4.4.3.12 does not appear to have a macOS .dmg file in the release. This is why it has not been updated in homebrew-cask yet. Once the .dmg is released I can submit the PR to get it updated for @Shawshank01.

Ah...sorry
https://github.com/c0re100/qBittorrent-Enhanced-Edition/releases/download/release-4.4.3.12/qBittorrent-Enhanced-Edition-release-4.4.3.12-macOS-universal.dmg

**Reply by p-linnane** [reply]
Thank you!

https://github.com/Homebrew/homebrew-cask/pull/125521
